src/plugins.js

```diff
@@ -271,7 +272,10 @@ export default class Plugins {
     const pjson = path.join(this.userPluginsDir, 'package.json')
     const yarnrc = path.join(this.userPluginsDir, '.yarnrc')
     fs.mkdirpSync(this.userPluginsDir)
-    if (!fs.existsSync(pjson)) fs.writeFileSync(pjson, JSON.stringify({private: true}))
```

Comment:
@dickeyxxx I agree with the approach, but it appears to me that this only runs on `plugins:install` unless I am missing something